frontend/src/v5/ui/routes/dashboard/projects/containers/containersList/containersList.helpers.ts

```diff
@@ -15,54 +15,69 @@
-	{
-		key: 1,
-		title: formatMessage({ id: 'containers.ellipsisMenu.loadContainer', defaultMessage: 'Load Container in 3D Viewer' }),
-		to: `/${id}`,
-	},
-	{
-		key: 2,
-		title: formatMessage({ id: 'containers.ellipsisMenu.uploadNewRevision', defaultMessage: 'Upload new Revision' }),
-		onClick: () => { },
-	},
-	{
-		key: 3,
-		title: formatMessage({ id: 'containers.ellipsisMenu.viewIssues', defaultMessage: 'View Issues' }),
-		onClick: () => { },
-	},
-	{
-		key: 4,
-		title: formatMessage({ id: 'containers.ellipsisMenu.viewRisks', defaultMessage: 'View Risks' }),
-		onClick: () => { },
-	},
-	{
-		key: 5,
-		title: formatMessage({ id: 'containers.ellipsisMenu.viewRevisions', defaultMessage: 'View Revisions' }),
-		onClick: () => { },
-	},
-	{
-		key: 6,
-		title: formatMessage({ id: 'containers.ellipsisMenu.editPermissions', defaultMessage: 'Edit Permissions' }),
-		onClick: () => { },
-	},
-	{
-		key: 7,
-		title: formatMessage({ id: 'containers.ellipsisMenu.shareContainer', defaultMessage: 'Share Container' }),
-		onClick: () => { },
-	},
-	{
-		key: 8,
-		title: formatMessage({ id: 'containers.ellipsisMenu.settings', defaultMessage: 'Settings' }),
-		onClick: () => {
+export const getContainerMenuItems = (id: IContainer['_id'], name: IContainer['name']) => {
+	const { teamspace, project } = useParams() as { teamspace: string, project: string };
+	const dispatch = useDispatch();
+
+	return [
+		{
+			key: 1,
+			title: formatMessage({ id: 'containers.ellipsisMenu.loadContainer', defaultMessage: 'Load Container in 3D Viewer' }),
+			to: `/${id}`,
+		},
+		{
+			key: 2,
+			title: formatMessage({ id: 'containers.ellipsisMenu.uploadNewRevision', defaultMessage: 'Upload new Revision' }),
+			onClick: () => { },
+		},
+		{
+			key: 3,
+			title: formatMessage({ id: 'containers.ellipsisMenu.viewIssues', defaultMessage: 'View Issues' }),
+			onClick: () => { },
+		},
+		{
+			key: 4,
+			title: formatMessage({ id: 'containers.ellipsisMenu.viewRisks', defaultMessage: 'View Risks' }),
+			onClick: () => { },
+		},
+		{
+			key: 5,
+			title: formatMessage({ id: 'containers.ellipsisMenu.viewRevisions', defaultMessage: 'View Revisions' }),
+			onClick: () => { },
+		},
+		{
+			key: 6,
+			title: formatMessage({ id: 'containers.ellipsisMenu.editPermissions', defaultMessage: 'Edit Permissions' }),
+			onClick: () => { },
+		},
+		{
+			key: 7,
+			title: formatMessage({ id: 'containers.ellipsisMenu.shareContainer', defaultMessage: 'Share Container' }),
+			onClick: () => { },
+		},
+		{
+			key: 8,
+			title: formatMessage({ id: 'containers.ellipsisMenu.settings', defaultMessage: 'Settings' }),
+			onClick: () => {
+			},
+		},
+		{
+			key: 9,
+			title: formatMessage({ id: 'containers.ellipsisMenu.delete', defaultMessage: 'Delete' }),
+			onClick: () => {
+				dispatch(DialogsActions.open('delete', {
+					title: formatMessage({ id: 'deleteModal.title', defaultMessage: `Delete ${name}?` }),
```

Comment:
For localization string interpolation is not used because they have to be already defined strings, thats why extract wont work with this one.
Formatjs has a way of allowing variables to be mixed up in the string and its like this:

```suggestion
					title: formatMessage({ id: 'deleteModal.title', defaultMessage: 'Delete {name}?' }, { name }),

```